src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,14 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: 'Under construction',
+};
+export const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+
+  },
```

Comment:
- [ ] Kindly add a reducer action here to check the status of your categories and make sure to export it as an action because it is the project requirement

![Screenshot from 2023-06-28 11-06-54](https://github.com/derrick1451/bookstore/assets/95412943/607d258b-7566-4e60-b51d-447f2530e15a)
: